packages/drawer/src/DrawerBase.tsx

```diff
@@ -24,7 +24,7 @@ export const DrawerBase = styled.div<Partial<DrawerProps> & { hiddenMargin: numb
     max-width: none;
     outline: none;
     overflow: hidden;
-    padding: ${rem(2)};
+    padding: ${rem(props.showCloseButton ? 6.8 : 2)} ${rem(2)} ${rem(2)};
```

Comment:
![image](https://github.com/little-spoon-dev/design-system/assets/6466844/91b1794c-6a42-409c-ab4d-0725894b1b91)

By default, when having the close button, you can see that it pushes the content down; this would work for that, but wouldn't it also push the cover image down if we were to use one?

Reply:
Yes, it would push the cover image down but for solving that 1) I reset the padding of the Drawer main container (see https://github.com/little-spoon-dev/little-spoon-frontend/blob/795fd92e634abfa4052c7d6dd18f2c7089aeed6b/app/components/utility/Drawer/DrawerWithCoverImage/styled.js#L7-L14) and add a container (with padding) for the content of the cover image drawer (see https://github.com/little-spoon-dev/little-spoon-frontend/blob/795fd92e634abfa4052c7d6dd18f2c7089aeed6b/app/components/utility/Drawer/DrawerWithCoverImage/styled.js#L44-L54)

The truth is that when I originally implemented the `Drawer` component I asked Chantal if it would always have the same layout/structure and the answer was yes. The new drawers with the cover image demonstrates that was not true.

Anyways, there could be another solution to all this problems but that would require a bigger refactoring.

Let me know how do you want me to procceed.

Reply:
I think this would be a common use case, so let's solve it in this component instead of having to override things in the FE for this to work.

Is there a reason not to use the suggested approach on the ticket? Unless I'm still missing something, It would require very little effort, be backward compatible, and unlock the option to keep the button positioned where it should, even if we place a cover image.

Let's go with the `closeButtonPosition` prop and keep the `data-test` and `aria-label` you added. If there are drawbacks or blockers with this approach, let's sync to figure out how to deal with them.

I think the only potential issue I forsee is the z-index, we might need to set it for both the tab content and the close button to avoid conflicts.